Chap_Terms.tex

```diff
@@ -4,39 +4,30 @@
 \chapter{PMIx Terms and Conventions}
 \label{chap:terms}
 
-The \ac{PMIx} Standard has adopted the widespread use of key-value \textit{attributes} to add flexibility to the functionality expressed in the existing \acp{API}. Accordingly, the community has chosen to require that the definition of each standard \ac{API} include the passing of an array of attributes. These provide a means of customizing the behavior of the \ac{API} as future needs emerge without having to alter or create new variants of it. In addition, attributes provide a mechanism by which researchers can easily explore new approaches to a given operation without having to modify the \ac{API} itself.
+In this chapter we describe some common terms and conventions used throughout this document.
+The \ac{PMIx} Standard has adopted the widespread use of key-value \textit{attributes} to add flexibility to the functionality expressed in the existing \acp{API}. Accordingly, the \ac{ASC} has chosen to require that the definition of each standard \ac{API} include the passing of an array of attributes. These provide a means of customizing the behavior of the \ac{API} as future needs emerge without having to alter or create new variants of it. In addition, attributes provide a mechanism by which researchers can easily explore new approaches to a given operation without having to modify the \ac{API} itself.
```

Comment:
Technical question.
This line seems to be unchanged. It's strange that GH shows it as deletion/addition.
Or am I missing something?

Reply:
Nevermind, I found the differences